tools/SourceKit/lib/SwiftLang/SwiftLangSupport.cpp

```diff
@@ -295,6 +296,45 @@ SwiftLangSupport::makeCodeCompletionMemoryBuffer(
   return std::unique_ptr<llvm::MemoryBuffer>(newBuffer.release());
 }
 
+bool SwiftLangSupport::setupCompilerInstanceForCodeCompletion(
```

Comment:
Should this move to `lib/IDE` so `swift-ide-test` can also use it and ensure that `swift-ide-test` performs code-completion the same way as `sourcekitd`?